source/configure/site-configuration-settings.rst

```diff
@@ -1715,3 +1716,71 @@ Enable end user notices
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------+
+
+Connected workspaces
+---------------------
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+The following settings aren't available in the System Console and can only be set in ``config.json``. 
+When connected workspaces are enabled, system admins can :doc:`create and manage connected workspaces </onboard/connected-workspaces>` in the System Console by going to **Site Configuration > Connected Workspaces (Beta)**.
+
+.. config:setting:: enable-connected-workspace
+  :displayname: Enable connected workspaces (Beta)
+  :systemconsole: Site Configuration > Connected Workspaces (Beta)
+  :configjson: ConnectedWorkspacesSettings.EnableSharedChannels, ConnectedWorkspacesSettings.EnableRemoteClusterService 
+  :environment: N/A
+  :description: Establish secure connections between Mattermost instances, and invite secured connections to shared channels
+
+Enable connected workspaces
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enable the ability to establish secure connections between Mattermost instances, and invite secured connections to shared channels where users can participate as they would in any public and private channel. 
+Both configuration settings are disabled by default and must be enabled in order to share channels with secure connections. Enabling connected workspace functionality requires a server restart.
+
+This feature's two ``config.json`` settings include:
+
+- ``ConnectedWorkspacesSettings.EnableRemoteClusterService: false`` with options ``true`` and ``false``.
+- ``ConnectedWorkspacesSettings.EnableSharedChannels: false`` with options ``true`` and ``false``.
+
+.. note::
+
+  - Neither setting is available in the System Console and can only be set in ``config.json``.
+  - System admins for Cloud deployments can submit a request to have these required configuration settings enabled for their Cloud deployment instance.
+
+.. config:setting:: disable-status-sync
+  :displayname: Disable shared channel status sync
+  :systemconsole: N/A
+  :configjson: ConnectedWorkspacesSettings.DisableSharedChannelsStatusSync
+  :environment: N/A
+
+  - **true**: **(Default)** Channel as well as channel member status and availability is synchronized at regular intervals.
+  - **false**: Channel as well as member status and availability isn't synchronized.
+
+Disable shared channel status sync
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Disable channel as well as member status and availability synchronization between connected workspaces.
```

Comment:
This configuration parameter only affects member status sync, I think we can remove the channel reference here and in the rest of the instances

```suggestion
Disable member status and availability synchronization between connected workspaces.
```